.github/workflows/ci.yml

```diff
@@ -35,8 +35,7 @@ jobs:
       - name: install dependencies
         run: yarn install --frozen-lockfile
 
-      - run: yarn lint:js
-
+      - run: yarn lint
```

Comment:
Isn't this duplicating some work since `yarn test` already runs lint?

Reply:
Good catch! Updated...

(I have another branch coming that makes two CI jobs, one for linting one for tests here and I mixed them up 😩)